src/sentry/api/event_search.py

```diff
@@ -360,6 +372,15 @@ def visit_numeric_filter(self, node, children):
             )
             return self._handle_basic_filter(search_key, "=", search_value)
 
+    def visit_result_filter(self, node, children):
+        (search_key, _, operator, search_value) = children
+        operator = operator[0] if not isinstance(operator, Node) else "="
+        try:
+            search_value = SearchValue(int(search_value.text))
```

Comment:
Will this be true for aggregates that operate on datetime columns?

Reply:
I added support for date aggregates although I really don't think anyone will use them.